cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -1978,22 +1978,22 @@ def Length(s,ctx=None):
     return ArithRef(ctx.solver.mkTerm(Kind.SEQ_LENGTH,s.ast),ctx)
 
 def SubString(s,offset,length,ctx=None):
-    """Extract substring or subsequence starting at offset"""
+    """Extract substring starting at offset"""
     ctx = _get_ctx(ctx)
     s = _py2expr(s)
     offset = _py2expr(offset)
     length = _py2expr(length)
     return StringRef(ctx.solver.mkTerm(Kind.STRING_SUBSTR,s.ast,offset.ast,length.ast),ctx)
 
 def SubSeq(s, offset, length):
+    """Extract substring or subsequence starting at offset"""
```

Comment:
should this include `substring`? not only `subsequence`?

Reply:
It works on strings as well, but I`ll change it anyway